### ISSUE TITLE
Error: Cannot find module 'redux-form'

### ISSUE THREAD

**Issue by eliseumds** [erikras/redux-form#4293]
### Sandbox Link

https://codesandbox.io/s/18qjp1jn8l

### What's your environment?

NodeJS v11.1.0

**Comment by gustavohenke** [comment]
That's strange -- it works for me if I download the sandbox and install it locally.

**Comment by dereknelson** [comment]
@eliseumds fixed the issue by doing the [fix here](https://github.com/erikras/redux-form/issues/4294#issuecomment-446400256)

**Comment by menewman** [comment]
We are seeing a very similar issue when attempting to update to react-redux 6.0.0 and redux-form 8.0.1, in a Meteor application that previously ran react-redux 5.1.1 and redux-form 7.4.2 without issue.

**Versions:**
Node 8.11.4
NPM 6.4.1
Meteor 1.8.0

It results in an error that looks like this:
```
Uncaught Error: Cannot find module 'redux-form'
    at makeMissingError (modules-runtime.js:232)
    at Module.require (modules-runtime.js:251)
    at Module.moduleLink [as link] (modules.js:355)
```
...when we try to do this:
```js
import { reducer as formReducer } from 'redux-form';
```

**Comment by dereknelson** [comment]
@menewman In the package.json change the main key from "main": "./index.js", to "main": "./es/index.js" and it'll work. I've already submitted a PR that fixes that, just waiting for it to be approved.

**Comment by menewman** [comment]
@dereknelson - Thanks for the tip. I don't want to fork this package in order to upgrade (and editing files in node_modules directly is no bueno), so we will just wait to upgrade a new version is published.

**Comment by dereknelson** [comment]
@menewman yeah I just did that because my app wouldn't start if form was broken. you can use https://www.npmjs.com/package/patch-package to patch-packages, comes in handy sometimes. (it only patches versions so you wouldn't have to worry about forward compatibility)

**Comment by menewman** [comment]
Haha, our production app probably shouldn't live on an edge that's quite that bleeding. But thanks for that tip, too; I didn't know about `patch-package` before!

**Comment by dereknelson** [comment]
I meant locally in development, I like to move fast and break things but not that fast lol
